### PR TITLE
Made a configurable constant

### LINKED ISSUE #1690: Make init intro msg a configurable constant
Make it a configurable constant.

_Originally posted by @sonichi in https://github.com/microsoft/autogen/pull/961#discussion_r1490464619_

### PR DESCRIPTION
**Pull request by RohitRathore1**
## Why are these changes needed?

This modification ensure that the introduction message is easily configurable and can be changed without altering the method's logic directly.

## Related issue number

Closes #1690 

## Checks

- [ ] I've included any doc changes needed for https://microsoft.github.io/autogen/. See https://microsoft.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [ ] I've made sure all auto checks have passed.

### CODE REVIEW

**Comment by JieyuZ2** [comment]
would it be better to make the intro msg an argument instead of configurable constant?

File: autogen/agentchat/groupchat.py [hunk 1]
```diff
@@ -236,10 +241,11 @@ def introductions_msg(self, agents: Optional[List[Agent]] = None) -> str:
         if agents is None:
             agents = self.agents
 
-        return f"""Hello everyone. We have assembled a great team today to answer questions and solve tasks. In attendance are:
+        # USe the class attribute instead of a hardcoded string
```
Comment (joshkyh) [inline autogen/agentchat/groupchat.py:244]:
```suggestion
        # Use the class attribute instead of a hardcoded string
```

**Comment by RohitRathore1** [comment]
> would it be better to make the intro msg an argument instead of configurable constant?

It would be a better option but as per issue I had opened my PR. @sonichi what do you think about the suggestion from @JieyuZ2? And other changes look good to me.

File: autogen/agentchat/groupchat.py (hunk 1 above)
Comment (RohitRathore1) [reply autogen/agentchat/groupchat.py]: Thanks!

**Review by sonichi — APPROVED** [review]

**Comment by sonichi** [comment]
> > would it be better to make the intro msg an argument instead of configurable constant?
> 
> It would be a better option but as per issue I had opened my PR. @sonichi what do you think about the suggestion from @JieyuZ2? And other changes look good to me.

@JieyuZ2 's suggestion can be addressed in a future PR if there are such needs.

**Pull request merged**